packages/opal-common/opal_common/schemas/data.py

```diff
@@ -35,7 +43,7 @@ class DataSourceConfig(BaseModel):
     updates)
     """
 
-    entries: List[DataSourceEntry] = Field(
+    entries: List[DataSourceEntryWithPollingInterval] = Field(
```

Comment:
@orishavit I didn't want to add it to the DataSourceEntry base class because that would imply that you could set the Pulling Interval on requested data changes made by something like a data update REST call tot he server. This keeps pulling interval siloed as only a configuration var.